workers/migrations/0015-create_default_indexes.js

```diff
@@ -78,6 +78,15 @@ var indexes = {
       "_id": 1
     },
     "ns": "koding.jStackTemplates"
+  }, {
```

Comment:
I know how migration works :) I thought since we didn't use it on prod db yet we don't need to pollute the migration stack and therefore it will work for dev environments since we're resetting db each time when we do `./run buildservices`, so it will work in both cases.